### PR TITLE
Heroku button

### LINKED ISSUE #4: Heroku button
It'd be awesome to have a [Heroku Button](https://devcenter.heroku.com/articles/heroku-button) to launch this easily on Heroku.

### PR DESCRIPTION
**Pull request by mwarkentin**
Fixes #4 

Adds a "Deploy to Heroku" button to the readme, along with the required `app.json` file.

### CODE REVIEW

**Comment by vjeux** [comment]
Could you rebase your pull request :) The additional info in app.json seems useful

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # mention-bot
 
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```
Comment (vjeux) [inline README.md]: The button is at the bottom, can you remove it here? I don't think that it should be that prominent

File: app.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+  "name": "mention-bot",
+  "description": "Automatically mention potential reviewers on pull requests.",
   "repository": "https://github.com/facebook/mention-bot",
-  "logo": "https://avatars3.githubusercontent.com/u/15710697?v=3&s=400"
-}
+  "logo": "https://avatars0.githubusercontent.com/u/15710697",
+  "keywords": ["node", "express", "github", "pull requests"],
+  "env": {
+    "GITHUB_TOKEN": {
+      "description": "A token generated for the account which will comment on your pull requests.",
```
Comment (vjeux) [inline app.json]: nice!

File: README.md (hunk 1 above)
Comment (mwarkentin) [inline README.md]: @vjeux Done!

**Comment by mwarkentin** [comment]
@vjeux Ok, I think everything's there now.

**Pull request merged**